hypertrace-trace-enricher/hypertrace-trace-enricher/src/main/resources/configs/common/application.conf

```diff
@@ -100,6 +100,10 @@ enricher {
   SpaceEnricher {
     class = "org.hypertrace.traceenricher.enrichment.enrichers.space.SpaceEnricher"
   }
+
+  ExitCallsEnricher {
```

Comment:
Add `dependencies`, if you need service information,
run at the end after `backend entity enricher`. if not, we can run after `ApiBoundaryTypeAttributeEnricher`

Reply:
Looked ExitCallsEnricher which needs backend and service name, so run this enricher after `BackendEntityEnricher`.

Reply:
done